librz/util/table.c

```diff
@@ -1179,7 +1212,39 @@ RZ_API void rz_table_visual_list(RzTable *table, RzList *list, ut64 seek, ut64 l
 		}
 	}
 }
+RZ_API RzTable *rz_table_transpose(RzTable *t) {
```

Comment:
Missing newline before. Please also, could you please add `RZ_OWN` marker for the return?
Please also add `rz_return_val_if_fail()` and `RZ_NONNULL`

Reply:
I recommend to add also the unit test for the transpose function